src/views/ContinuationIn/ContinuationInReviewConfirm.vue

```diff
@@ -183,7 +196,7 @@
 
     <!-- Staff Payment -->
     <section
-      v-if="isRoleStaff"
+      v-if="isRoleStaff && getFilingStatus === FilingStatus.DRAFT"
```

Comment:
Don't show staff payment section if the filing is Change Requested. You'll see in filing-template-mixin that we hard-code the filing data to "waive fees" if we're resuming a Change Requested filing. This sets the Fee Summary to $0, ie:

![image](https://github.com/user-attachments/assets/c5c058cb-c70c-4875-ba03-60c13dbc5965)